src/runtime/HalideRuntime.h

```diff
@@ -19,6 +19,12 @@ template<typename T> struct halide_handle_traits;
 extern "C" {
 #endif
 
+#ifdef __LP64__                           /* DMA Specific Type For Address */
```

Comment:
This is an unqualified name (e.g. it does not begin with "halide_") in the global address space in a widely used header. This name is likely to conflict with a hundred things. The declaration uses an #ifdef for choosing the type that we likely would not rely on here. There is no documentation and it probably doesn't want to be in this header file anyway.